timesketch/api/v1/resources/explore.py

```diff
@@ -156,15 +156,23 @@ def post(self, sketch_id: int):
 
         all_indices = list({t.searchindex.index_name for t in sketch.timelines})
         indices = query_filter.get("indices", all_indices)
+        all_timeline_ids = [t.id for t in sketch.timelines]
```

Comment:
This does not take into account if a timeline is ready to be searched, correct? So it would also return timeline_ids that are still in the processing status. This can lead to unexpected behaviour as well since we would return partial results for a timeline. We probably need something along the lines of
```
if timeline.get_status.status.lower() != "ready":
            continue
```

Reply:
it does not, but the line later

```
        indices = utils.validate_indices(indices, self.datastore)
```

is checking that, so I think we are fine